### ISSUE TITLE
Versão com plugin de detecção de renaming

### ISSUE THREAD

**Issue by pauloborba** [guilhermejccavalcanti/jFSTMerge#16]

**Comment by pauloborba** [comment]
testar alternativas para detectar renaming

**Comment by pauloborba** [comment]
@guilhermejccavalcanti, coloca @Arthurlpgc como colaborador. ele está estudando o RMiner, e vai tentar focar nesta issue

**Comment by pauloborba** [comment]
1. testar a RMiner
2. verificar possibilidade de integração (via jar, fonte, ?)
3. novo handler (melhorias nos de @gaabs? discutir bem o design)
4. integrar
5. avaliar

**Comment by guilhermejccavalcanti** [comment]
@pauloborba confirma o login por favor, tá dando inválido.

**Comment by pauloborba** [comment]
@arthurlpgc

**Comment by pauloborba** [comment]
clicando aí leva ao usuário

**Comment by guilhermejccavalcanti** [comment]
done.

**Comment by Arthurlpgc** [comment]
Update: Iniciei a task, e parece que vai ser bem simples de integrar(pelo menos pelo lado do RMiner)

**Comment by pauloborba** [comment]
@Arthurlpgc a rminer funcionou bem? em que projetos voce a testou? qual a ideia para integracao? via jar mesmo?

**Comment by pauloborba** [comment]
@Arthurlpgc você já olhou o handler de renaming? o mais básico usa a similaridade textual entre os corpos dos métodos para identificar renaming. @guilhermejccavalcanti podes apontar para isso, por favor? @gaabs podes apontar para os teus handlers também?

**Comment by guilhermejccavalcanti** [comment]
Aqui https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/MethodAndConstructorRenamingAndDeletionHandler.java#L122

**Comment by gaabs** [comment]
Os meus handlers, por hora, também estão contidos nesse que Guilherme citou. De acordo com a configuração do jFSTMerge, o devido fluxo é aplicado.

Já estou trabalhando em uma branch que os separa em handlers distintos. Quando for mergido, atualizo aqui.

**Comment by Arthurlpgc** [comment]
Obrigado @gaabs e @guilhermejccavalcanti, consegui encontrar aqui os arquivos, sobre o RMiner, ele parece funcionar bem, e a ideia é incluir o jar sim

**Comment by Arthurlpgc** [comment]
Acho que houveram uns refactors na parte do código onde eu estava mexendo, alguém pode confirmar?

**Comment by gaabs** [comment]
> Acho que houveram uns refactors na parte do código onde eu estava mexendo, alguém pode confirmar?

Sim, recentemente mergi uma modificação nesse handler. Não sei exatamente onde você estava mexendo, mas os handlers anteriores viraram esses dois:

1. https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/mutualrenaming/singlerenaming/DefaultMutualRenamingHandler.java
2. https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/singlerenaming/DefaultSingleRenamingHandler.java

Foi resultado desse PR: https://github.com/guilhermejccavalcanti/jFSTMerge/pull/66
